README.md

```diff
@@ -26,6 +26,7 @@ This log lists the architectural decisions for Trendyol iOS Project.
 - [ADR-0019](adr/0019-naming-branch-name.md) - Naming branch name
 - [ADR-0020](adr/0020-inject-theme-manager-for-common-places.md) - Inject theme manager for common places
 - [ADR-0021](adr/0021-testing-only-one-function.md) - Testing only one function for each test case
+- [ADR-0022](adr/0022-use-reflaction-for-unit-tests.md) - Use reflaction for unit tests
```

Comment:
`- [ADR-0022](adr/0022-use-reflection-for-unit-tests.md) - Use reflection for unit tests`